resources/oracle-db/INSTALL.md

```diff
@@ -29,16 +29,6 @@ A file with custom queries for extra metrics can be found in the `custom-metrics
 
 # Sysdig Agent configuration
 
```

Comment:
This text is missing:
2. Download the sample [Sysdig Agent configuration file](include/sysdig-agent-config.yaml) and apply it by: